backend/handlers/users/usercredit.go

```diff
@@ -0,0 +1,64 @@
+package usershandlers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"socialpredict/setup"
+	"socialpredict/util"
+
+	"github.com/gorilla/mux"
+	"gorm.io/gorm"
+)
+
+// appConfig holds the loaded application configuration accessible within the package
+var appConfig *setup.EconomicConfig
+
+func init() {
+	var err error
+	appConfig, err = setup.LoadEconomicsConfig()
+	if err != nil {
+		log.Fatalf("Failed to load configuration: %v", err)
+	}
+}
+
+type UserCreditResponse struct {
+	Credit int `json:"credit"`
+}
+
+// for usage on sidebar or continuously throughout application in order to continuously show available spend
+func GetUserCreditResponse(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function implements the [handler interface](https://pkg.go.dev/net/http#Handler), I'd recommend renaming it to `GetUserCreditHandler`. It only supports `GET` requests, so the Get Prefix is descriptive of how to use this handler can be used.

Reply:
Resolved. Should go in styleguide because a lot of other functions used in the server are named `Response` rather than `Handler`.